app/models/form_item.rb

```diff
@@ -39,7 +39,7 @@ def _sorted_leaves(nodes)
         if form_item.is_a?(Questioning)
           form_item
         else
-          _sorted_leaves(children)
+          _sorted_leaves(children).flatten
```

Comment:
Will this prohibit multiple nesting in general? We don't want to do that I think. Only for now when we're rendering for ODK. The user should still be able to nest multiply when defining a form. There are other reasons this might be needed.